server/models/City.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require ("mongoose")
+const Schema = mongoose.Schema
+
+const citySchema = new Schema({
+    name: String,
+    temperature: Number,
+    condition: String,
+    conditionPic: String
+  
+})
+const Cities = mongoose.model("city", citySchema)
```

Comment:
The variable name should be the same as the first argument in the `model` method. So here they should both be `City`.